From 9a3888b7013d5cca251b0a63585fd211362e9eac Mon Sep 17 00:00:00 2001
From: Saurav Shah <quic_sauravs@quicinc.com>
Date: Mon, 10 Jun 2024 20:46:59 +0530
Subject: [PATCH] qdl: Change usb macro

Change usb macro to fix compilation
issues.

This patch file is from Qualcomm Innovation Center, Inc. and is provided under the following license:

Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
SPDX-License-Identifier: BSD-3-Clause-Clear 

Signed-off-by: Saurav Shah <quic_sauravs@quicinc.com>
---
 usb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usb.c b/usb.c
index 35a1c1e..fbdce7a 100644
--- a/usb.c
+++ b/usb.c
@@ -81,7 +81,7 @@ static int qdl_try_open(libusb_device *dev, struct qdl_device *qdl, const char *
 			endpoint = &ifc->endpoint[l];
 
 			type = endpoint->bmAttributes & LIBUSB_TRANSFER_TYPE_MASK;
-			if (type != LIBUSB_ENDPOINT_TRANSFER_TYPE_BULK)
+			if (type != LIBUSB_TRANSFER_TYPE_BULK)
 				continue;
 
 			if (endpoint->bEndpointAddress & LIBUSB_ENDPOINT_IN) {
-- 
2.17.1

